crates/task-impls/src/consensus.rs

```diff
@@ -453,15 +414,15 @@ where
                         error!("Block payload commitment does not equal parent commitment");
                         return false;
                     }
-                    let vote = QuorumVote::<TYPES, QuorumMembership<TYPES, I>>::create_signed_vote(
+                    let vote = QuorumVote::<TYPES>::create_signed_vote(
                         QuorumData {
                             leaf_commit: leaf.commit(),
                         },
                         view,
-                        self.quorum_exchange.public_key(),
-                        self.quorum_exchange.private_key(),
+                        &self.public_key,
+                        &self.private_key,
```

Comment:
I feel like we'll need to have public and private key as fields on every task.  Would it make sense to make them part of the Membership type so we don't have to pass them in as fields all the time?